zerver/tests/test_realm_linkifiers.py

```diff
@@ -109,8 +109,55 @@ def test_delete(self) -> None:
         )
         linkifiers_count = RealmFilter.objects.count()
         result = self.client_delete(f"/json/realm/filters/{linkifier_id + 1}")
-        self.assert_json_error(result, "Filter not found")
+        self.assert_json_error(result, "Linkifier not found.")
 
         result = self.client_delete(f"/json/realm/filters/{linkifier_id}")
         self.assert_json_success(result)
         self.assertEqual(RealmFilter.objects.count(), linkifiers_count - 1)
+
+    def test_update(self) -> None:
+        self.login("iago")
+        realm = get_realm("zulip")
+        linkifier_id = do_add_linkifier(
+            realm, "#(?P<id>[123])", "https://realm.com/my_realm_filter/pull/%(id)s"
+        )
+        result = self.client_get("/json/realm/linkifiers")
+        self.assert_json_success(result)
+
+        do_update_linkifier(
+            realm,
+            linkifier_id,
+            "#(?P<id>[0-9]+)",
+            "https://realm.com/my_realm_filter/issues/%(id)s",
+        )
+        result = self.client_get("/json/realm/linkifiers")
+        self.assert_json_success(result)
+        self.assertEqual(200, result.status_code)
+        self.assertEqual(len(result.json()["linkifiers"]), 1)
```

Comment:
You should verify the value returned here, to make sure it was actually updated.